src/main/java/fr/inria/coming/changeminer/analyzer/instancedetector/PatternInstancesFromDiff.java

```diff
@@ -57,10 +64,12 @@ public String toString() {
 			for (ChangePatternInstance instance : instances) {
 				resultString += "\n" + instance.toString();
 			}
-			return resultString += "\n----";
+            log.info(resultString += "\n----");
```

Comment:
undo this. a toString remains as is, we must remove the call to tostring instead.

Reply:
ping